### PR TITLE
Add examples for SE and MP to update counters of HTTP response status ranges (1xx, 2xx, etc.) for 3.x

### LINKED ISSUE #4582: Consider adding metrics examples for SE and MP showing how to count HTTP responses in each range: 1xx , 2xx, etc.
## Environment Details
* Helidon Version: 2.x, 3.x
* Helidon SE or Helidon MP both
* JDK version:
* OS:
* Docker version (if applicable):

----------

## Problem Description
We have had multiple users ask how to create metrics which report the number of HTTP responses in each family (1xx, 2xx, etc.)

This is relatively easily done by users so creating examples might be a good alternative vs. adding this as a core feature of Helidon metrics.

### PR DESCRIPTION
**Pull request by tjquinno**
Resolves #4582 for 3.x

### CODE REVIEW

File: examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        JsonObject returnObject = JSON.createObjectBuilder()
+                .add("message", msg)
+                .build();
+        response.send(returnObject);
+    }
+
+    private static <T> T processErrors(Throwable ex, ServerRequest request, ServerResponse response) {
+
+         if (ex.getCause() instanceof JsonException){
```
Comment (spericas) [inline examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java:108]: "{" -> " {"

File: examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+            LOGGER.log(Level.FINE, "Invalid JSON", ex);
+            JsonObject jsonErrorObject = JSON.createObjectBuilder()
+                .add("error", "Invalid JSON")
+                .build();
+            response.status(Http.Status.BAD_REQUEST_400).send(jsonErrorObject);
+        }  else {
+
```
Comment (spericas) [inline examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java:116]: Empty line after "{"?

File: examples/metrics/http-status-count-se/src/test/java/io/helidon/examples/se/httpstatuscount/StatusTest.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            webServer.shutdown()
+                    .toCompletableFuture()
+                    .get(10, TimeUnit.SECONDS);
+        }
+    }
+
+    @BeforeEach
+    void findStatusMetrics() {
+        metricRegistry = RegistryFactory.getInstance().getRegistry(MetricRegistry.Type.APPLICATION);
+        for (int i = 1; i < 6; i++) {
```
Comment (spericas) [inline examples/metrics/http-status-count-se/src/test/java/io/helidon/examples/se/httpstatuscount/StatusTest.java:77]: "< STATUS_COUNTERS.length"?

**Review by spericas — APPROVED** [review]
LGTM, some very minor comments.

File: examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java (hunk 1 above)
Comment (tjquinno) [reply examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java:108]: Fixed

File: examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java (hunk 2 above)
Comment (tjquinno) [reply examples/metrics/http-status-count-se/src/main/java/io/helidon/examples/se/httpstatuscount/GreetService.java:116]: Fixed

File: examples/metrics/http-status-count-se/src/test/java/io/helidon/examples/se/httpstatuscount/StatusTest.java (hunk 3 above)
Comment (tjquinno) [reply examples/metrics/http-status-count-se/src/test/java/io/helidon/examples/se/httpstatuscount/StatusTest.java:77]: Fixed

**Review by spericas — APPROVED** [review]

**Pull request merged**